### PR TITLE
Yl 377 move icons and logos from submodule to repo

### PR DESCRIPTION
**Pull request by mljubic-biz**

### CODE REVIEW

File: components/molecules/PasswordInput/PasswordInput copy.tsx
Comment (malispahic) [inline components/molecules/PasswordInput/PasswordInput copy.tsx:1]: Is this file needed since it is called `PasswordInput copy.tsx`

**Review by malispahic — APPROVED** [review]
One question for the file otherwise looks good.

File: components/molecules/PasswordInput/PasswordInput copy.tsx
Comment (mljubic-biz) [reply components/molecules/PasswordInput/PasswordInput copy.tsx:1]: This is a mistake, good catch!
